src/lib/api/configuration-shell-api/configurationShellApi.ts

```diff
@@ -16,7 +16,17 @@ export class ConfigurationShellApi {
         this.use_authentication = use_authentication;
     }
 
-    public async getIdGenerationSettings(): Promise<Submodel> {
+    static create(
```

Comment:
what about `createNull`?

Reply:
There is no createNull yet, because there are no tests yet for configuration shells. I think writing tests for all APIs we extracted would be to much for this ticket/story, because it would basically mean writing tests for ALL API classes.